code/controllers/configuration/entries/combat_defines.dm

```diff
@@ -122,55 +122,55 @@ Damage.
 Damage of the projectile, though the gun itself can modify it. This is the main source of projectile damage.
 */
 /datum/config_entry/number/combat_define/base_hit_damage
-	config_entry_value = 10
+	config_entry_value = 9
 
 /datum/config_entry/number/combat_define/min_hit_damage
-	config_entry_value = 15
+	config_entry_value = 13.5
```

Comment:
This kind of change is not going to pass, we want the combat defines to be more or less consistent, not to add another layer of obscurity to them.
The solution is to nerf specific weapons instead. More work, yes, but less random results.